README.md

```diff
@@ -20,13 +20,13 @@ Orbs are packages of CircleCI configuration that can be shared across projects.
 You can find additional documentation detailing orbs, including how to use and create them, in [this repo](https://github.com/CircleCI-Public/config-preview-sdk/tree/master/docs).
 
 ## Publishing Orbs
-Orbs are added to the registry through the CircleCI API. The build in this repository (`.circleci/config.yml`) uses that API via the [CircleCI CLI](https://github.com/CircleCI-Public/circleci-cli) to take the source of orbs located in the `src` folder and register them as orbs.
+Orbs are added to the registry through the CircleCI API. The build-in this repository (`.circleci/config.yml`) uses that API via the [CircleCI CLI](https://github.com/CircleCI-Public/circleci-cli) to take the source of orbs located in the `src` folder and register them as orbs.
```

Comment:
This is not correct, your other change is fine.

Reply:
I have corrected the readme, according to the instructution.